lib/mumuki/laboratory/locales/es.yml

```diff
@@ -202,6 +202,7 @@ es:
   read: Leido
   read_messages: Ver mensajes
   refresh_or_wait: Si no se muestra automáticamente en unos segundos, presioná F5
+  requires_attention: Requiere atención
   reset_query: Borrar los filtros de búsqueda actuales
   responses_count_asc: Con menos respuestas de moderadores
```

Comment:
Let's change it for "respuestas validadas" instead of "respuestas de moderadores"